metrics.go

```diff
@@ -65,7 +67,8 @@ type Metric struct {
 // Metrics contains the metrics declarations encoded in the metrics.yaml
 // file.
 type Metrics struct {
-	Metrics map[string]Metric
+	Metrics map[string]Metric `yaml:"metrics"`
+	Plan    string            `yaml:"plan"`
```

Comment:
Good point! Yes, I think something like:
```
plan:
  required: true|false
```
is just as clear and concise, but more open.